spec/features/cart_spec.rb

```diff
@@ -32,6 +32,9 @@
     page.find(:css, '.cart-link').click
     page.find(:css, '#cart-aside .remove').click
 
+    visit sprangular_engine.root_path(anchor: "!/products/#{mug.slug}")
+    page.find(:css, '.add-to-cart').click
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.